chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -387,6 +387,52 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
             dispatch({ type: LiveChatWidgetActionType.SET_MINIMIZED, payload: msg?.payload?.minimized });
         });
 
+
+
+        BroadcastService.getMessageByEventName("sendCustomEvent").subscribe((event: object) => {
+            let payload = {};
```

Comment:
this logic is not using at all states, I think it will be better to move it to its own file as a hook,